ethcore/src/service.rs

```diff
@@ -36,6 +36,8 @@ pub enum SyncMessage {
 		retracted: Vec<H256>,
 		/// Hashes of blocks that are now included in cannonical chain
 		enacted: Vec<H256>,
+		/// Hashes of blocks that are sealed by this node
+		sealed: Vec<H256>,
```

Comment:
`Option<H256>` should be enough here, no? Could simplify couple of other places. But I'm good with `Vec` for constistency